R/install.R

```diff
@@ -57,7 +57,8 @@ safe_install_packages <- function(...) {
   with_envvar(
     c(R_LIBS = lib,
       R_LIBS_USER = lib,
-      R_LIBS_SITE = lib),
+      R_LIBS_SITE = lib,
+      RGL_USE_NULL = TRUE),
```

Comment:
These are strings, so can you please use `"TRUE"`? (I know that it works with `TRUE` as well.)

Reply:
TRUE. Done!